public/editor/scripts/project/project.js

```diff
@@ -246,12 +247,33 @@ define(function(require) {
 
             // Look for an HTML file to open, ideally index.html
             var indexPos = 0;
+            var foundIndexHTML = false;
             found.forEach(function(path, idx) {
               if(Path.basename(path) === "index.html") {
                 indexPos = idx;
+                foundIndexHTML = true;
               }
             });
 
+            // Create a default index.html file
+            if (!foundIndexHTML) {
+              var location = "/default-files/html.txt";
+              $.get(location).then(function(data) {
+                var file = getRoot() + "/index.html";
+                _fs.writeFile(file, data, function(err) {
+                  if (err) {
+                    return console.error("Cannot write file to project: " + err);
+                  }
```

Comment:
I just re-tested the code and noticed that when I had the `callback()`, it created an error in the console like this:
<img width="640" alt="screen shot 2017-03-14 at 10 22 25 am" src="https://cloud.githubusercontent.com/assets/10606265/23915708/3bf3b424-08c0-11e7-80b0-035b738ef81b.png">

With this screen showed up after the thimble loading screen for a split second and it disappeared:
<img width="787" alt="screen shot 2017-03-14 at 11 21 57 am" src="https://cloud.githubusercontent.com/assets/10606265/23915872/b9a76226-08c0-11e7-9947-1a94ecac3dd4.png">

Therefore, I don't think we need a `callback()` here. The code will still work without it.